deployments/kubernetes/chart/reloader/templates/deployment.yaml

```diff
@@ -86,6 +86,19 @@ spec:
         image: "{{ .Values.reloader.deployment.image.name }}:{{ .Values.reloader.deployment.image.tag }}"
         imagePullPolicy: {{ .Values.reloader.deployment.image.pullPolicy }}
         name: {{ template "reloader-fullname" . }}
+
+        ports:
+        - name: http-metrics
+          containerPort: 9090
```

Comment:
Can you please templatize the container port from the values.yaml file.

Reply:
Not trying to step on toes, but I noticed that the metrics port in the go code is hardcoded.
So does it really make sense to template it?

https://github.com/stakater/Reloader/blob/39b5be37af3eb87c54430ebe0dc0ed801ed377a6/internal/pkg/metrics/prometheus.go#L39

Reply:
You are right, the port in the code should not be hardcoded, a new PR can be opened for it. For now, the port is already templatized in the chart for service.yaml so it is good to use the same variable in both the deployment and service for consistency and control the port in service and deployment from one common place in values.yaml.

Reply:
true true :-)

Reply:
* port of **service** templatized
* port of **pod** hardcoded

For consistency I replace `reloader.service.ports` array in values to integer value `reloader.service.port`